CHANGELOG_temp.md

```diff
@@ -0,0 +1,37 @@
+Temporary changelog for the long-lived branch for decK-apiops. To minimize conflicts
```

Comment:
Can we please remove this and add a `v1.24.0` entry in the main Changelog?

Reply:
done